internal/server/iifo.go

```diff
@@ -73,13 +73,12 @@ func (s *SOMASServer) distributePredictions(islandPredictionDict shared.Predicti
 	}
 
 	// Now distribute these predictions to the islands
-	for id, ci := range s.gameState.ClientInfos {
-		if ci.LifeStatus != shared.Dead {
-			c := s.clientMap[id]
-			err = c.ReceivePredictions(receivedPredictionsDict[id])
-			if err != nil {
-				return errors.Errorf("Failed to receive prediction from client %v: %v", id, err)
-			}
+	nonDeadClients := getNonDeadClientIDs(s.gameState.ClientInfos)
+	for _, id := range nonDeadClients {
+		c := s.clientMap[id]
+		err = c.RecievePredictions(recievedPredictionsDict[id])
```

Comment:
typo in `recievedPredictionsDict`, and `c.RecievePredictions`. Checking if this exists anywhere else.

Reply:
Yall merged it while I was reviewing :(

Reply:
#94 fixed